### ISSUE TITLE
Transform intellij.java.decompiler into a gradle module

### ISSUE THREAD

**Issue by AdlerFleurant** [AdlerFleurant/intellij-community#35]

**Comment by AdlerFleurant** [comment]
Depends on #30  #132 #133 #19 #10 #87 #131

**Comment by AdlerFleurant** [comment]
Won't fix as this is a plugin.
